src/main.js

```diff
@@ -0,0 +1,145 @@
+import { PanelCtrl } from 'app/plugins/sdk';
```

Comment:
wrong file name:
should be AC2Ctrl.js instead of main.js

Reply:
done